content/setup/local-with-vagrant/_index.md

```diff
@@ -0,0 +1,264 @@
+
+## Prerequisite
+
+- The host's processor should support virtualization
+- [Vagrant](https://www.vagrantup.com/downloads) is installed
+- Either [VirtualBox](https://www.virtualbox.org/) or [libvirtd](https://libvirt.org/) is installed and configured.
+
+## Cloning Tinkerbell
+
+1. Clone `tink` and move inside `deploy/vagrant` directory. This folder contains
+   vagrant configuration (Vagrantfile) and the scripts needed to provide workers
+   and provisioners.
+
+```
+$ git clone https://github.com/tinkerbell/tink.git
+$ cd tink/deploy/vagrant
+```
+
+## Start the provisioner
+
+```
+$ vagrant up provisioner
+Bringing machine 'provisioner' up with 'virtualbox' provider...
+==> provisioner: Importing base box 'generic/ubuntu1804'...
+==> provisioner: Matching MAC address for NAT networking...
+==> provisioner: Checking if box 'generic/ubuntu1804' version '3.0.10' is up to date...
+==> provisioner: Setting the name of the VM: tink-vagrant_provisioner_1592388207301_7361
+==> provisioner: Clearing any previously set network interfaces...
+==> provisioner: Preparing network interfaces based on configuration...
+    provisioner: Adapter 1: nat
+    provisioner: Adapter 2: hostonly
+==> provisioner: Forwarding ports...
+    provisioner: 22 (guest) => 2222 (host) (adapter 1)
+==> provisioner: Running 'pre-boot' VM customizations...
+==> provisioner: Booting VM...
+==> provisioner: Waiting for machine to boot. This may take a few minutes...
+    provisioner: SSH address: 127.0.0.1:2222
+    provisioner: SSH username: vagrant
+    provisioner: SSH auth method: private key
+    provisioner:
+    provisioner: Vagrant insecure key detected. Vagrant will automatically replace
+    provisioner: this with a newly generated keypair for better security.
+    provisioner:
+    provisioner: Inserting generated public key within guest...
+    provisioner: Removing insecure key from the guest if it's present...
+    provisioner: Key inserted! Disconnecting and reconnecting using new SSH key...
+==> provisioner: Machine booted and ready!
+==> provisioner: Checking for guest additions in VM...
+==> provisioner: Setting hostname...
+==> provisioner: Configuring and enabling network interfaces...
+==> provisioner: Rsyncing folder: /Users/name/git/tink/ => /vagrant
+==> provisioner: Running provisioner: shell...
+    provisioner: Running: /var/folders/0_/841vr1ms57s65r7h1_2vm69m0000gn/T/vagrant-shell20200617-5179-181phru.sh
+    provisioner: + whoami
+    provisioner: root
+    provisioner: + cd /vagrant
+    provisioner: + main
+    provisioner: + export DEBIAN_FRONTEND=noninteractive
+    provisioner: + DEBIAN_FRONTEND=noninteractive
+    provisioner: + apt-get update
+    provisioner: Hit:1 http://us.archive.ubuntu.com/ubuntu bionic InRelease
+    provisioner: Get:2 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
+    provisioner: Get:3 http://us.archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
```

Comment:
Do you need the whole output here or would an abridged version serve the same purpose and make the doc shorter?

Reply:
Heavily trimmed!